examples/OneDShallowWaterGeostrophicAdjustment.jl

```diff
@@ -1,184 +1,428 @@
+# # Linear rotating shallow water dynamics
+#
+#md # This example can be run online via [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/OneDShallowWaterGeostrophicAdjustment.ipynb). 
```

Comment:
Binder and Jupyter notebook links seem to be broken

Reply:
Yeap thanks for catching this; I moved the documentation in a different repository and I have to figure out what the new urls are...

Reply:
ad04e13 fixes this